### PR TITLE
Support overhaul

### PR DESCRIPTION
**Pull request by alexrolleyoctopus**
Adding Junior and Principal roles, along with a general updates to each role.

### CODE REVIEW

**Review by tmbam — APPROVED** [review]
Support Team role descriptions are well developed and build consistently on one another. The addition of Junior and Principle roles rounds out and completes the Team's growth lifecycle.

File: Support.md [hunk 1]
```diff
@@ -1,30 +1,67 @@
...
+
+_I am starting my journey at Octopus. I am an enthusiastic learner, I show great potential and have a high growth trajectory toward independent contribution._
+
+
+>We avoid distinguishing too much between Junior Support Engineer and Support Engineer. These roles are primarily focused on learning with the aim to grow into the contribution of a Senior Support Engineer or beyond. The primary indicator inside these levels is how far you are along your journey from dependent contribution to independent contribution.
+
+### I aim to write high quality responses quickly and with minimal oversight
+- I recognize Octopus is complex and I am starting on my knowledge journey, I reach out for help in a timely and effective way when I need it.
+- I am learning to write well crafted, succinct and informative support responses with coaching from my mentor.
+- I have started my learning journey in my area specialty at Octopus, I am seeking to be able to independently answer support questions in this area.
```
Comment (wordlee) [inline Support.md:13]:
```suggestion
- I have started my learning journey in my area speciality at Octopus, I am seeking to be able to independently answer support questions in this area.
```

File: Support.md [hunk 2]
```diff
@@ -1,30 +1,67 @@
...
+- I am becoming proficient with my tools and the core building blocks used by our team.
+- I am learning to prioritise my workload.
+- I am learning to recognise when I need help, and how to do this in an effective and timely manner.
+- I am a self-managing team member, I make and keep my commitments using the privilege of flexibility wisely.
+
+### I am recognised by my desire to learn and my motivation to improve
+
+- I take my individual growth seriously and realise it is valuable to both myself and to Octopus.
+
+- I take the privilege of Sharpening seriously, and work with my Manager to set realistic, achievable and relevant learning goals.
```
Comment (wordlee) [inline Support.md:24]:
```suggestion
- I take the privilege of Sharpening seriously, and work with my manager to set realistic, achievable and relevant learning goals.
```

File: Support.md [hunk 3]
```diff
@@ -1,30 +1,67 @@
...
+- I am an active learner. I use a variety of learning methods, like reading books, reading code, configuring test environments, watching online courses, attending conferences or user group meetings to help improve my knowledge of Octopus and surrounding technology.
+- I am discussing the things I'm learning about or experimenting on with my teammates and manager, maybe even collaborating on the experience.
+- I recognise I am my career owner. I reach out to my peers and my manager to develop my personal growth plan.
```
Comment (wordlee) [inline Support.md:27]:
```suggestion
- I recognise I own my career. I reach out to my peers and my manager to develop my personal growth plan.
```

File: Support.md [hunk 4]
```diff
@@ -1,30 +1,67 @@
...
+
+### I am an egoless, team-oriented support engineer
+- Other team members genuinely want to work alongside me.
+- I happily take responsibility for what I produce.
+- I readily accept input from others, and understand that good ideas can come from any source.
```
Comment (wordlee) [inline Support.md:32]:
```suggestion
- I readily accept input from others, and I understand that good ideas can come from any source.
```

File: Support.md [hunk 5]
```diff
@@ -1,30 +1,67 @@
...
 - I am learning to prioritise my workload.
 - I am learning to recognise when I need help, and how to do this in an effective and timely manner.
 - I am a self-managing team member, I make and keep my commitments using the privilege of flexibility wisely.
 
 ### I am recognised by my desire to learn and my motivation to improve
-
-- I am learning, maybe through a variety of ways like reading books, reading code, configuring test environments, watching online courses, attending conferences or user group meetings.
-- I am discussing the things I'm learning about or experimenting on with my teammates, maybe even collaborating on the experience.
 - I take my individual growth seriously and realise it is valuable to both myself and to Octopus.
+- I take the privilege of Sharpening seriously, and work with my Manager to set realistic, achievable and relevant learning goals.
```
Comment (wordlee) [inline Support.md:59]:
```suggestion
- I take the privilege of Sharpening seriously, and work with my manager to set realistic, achievable and relevant learning goals.
```

File: Support.md [hunk 6]
```diff
@@ -76,41 +110,67 @@ The Senior Support Engineer role builds upon the foundation of the Support Engin
...
 
-> I am a technical leader, a person that is recognised both within the Support team and as part of the wider Octopus community for the breadth and depth of your technical ability.
+_The Lead Support Engineer role builds upon the foundation of the earlier roles._
 
 ### I am trusted
-
 - I have mastered my tools and the core building blocks used by our team.
-- I have a deep technical understanding of more than one area of Octopus, and I can confidently use this knowledge to assist customers and fellow support team members in all areas of Octopus.
-- I know when it makes sense to push back against a customer, and when it is appropriate to not create an issue
+- I have a deep technical understanding of my area of specialty at Octopus, and I can confidently use this knowledge to assist customers and fellow support team members in all areas of Octopus.
```
Comment (wordlee) [inline Support.md:126]:
```suggestion
- I have a deep technical understanding of my area of speciality at Octopus, and I can confidently use this knowledge to assist customers and fellow support team members in all areas of Octopus.
```

File: Support.md [hunk 7]
```diff
@@ -76,41 +110,67 @@ The Senior Support Engineer role builds upon the foundation of the Support Engin
...
 - I am constantly learning new technologies and methodologies that relate to Octopus.
 - I work with teams outside of Support to further enhance my knowledge
 - I happily take responsibility for my own mistakes and I learn from the mistakes of others.
 - I share the lessons I learn so the entire team and Octopus can benefit
+
+
+
+## Principal Support Engineer
+
+_I am an influential Support Leader, a person who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Support to deliver significant lasting change across Octopus._
```
Comment (wordlee) [inline Support.md:155]: I like this distinction. Really helps explain the influence component of thinking and acting in a much broader way.

File: Support.md [hunk 8]
```diff
@@ -76,41 +110,67 @@ The Senior Support Engineer role builds upon the foundation of the Support Engin
...
+
+_The Principal Support Engineer role builds upon the foundation of the earlier Support roles._
+
+### I uncover important problems and opportunities that are critical to the long-term success of Octopus
+- I have a deep understanding of my area of specialty, as well as Octopus products and strategy
```
Comment (wordlee) [inline Support.md:160]:
```suggestion
- I have a deep understanding of my area of speciality, as well as Octopus products and strategy
```

File: Support.md [hunk 9]
```diff
@@ -76,41 +110,67 @@ The Senior Support Engineer role builds upon the foundation of the Support Engin
...
+- I collaborate with other Octopus leaders to ensure I am aware of all major initiatives at Octopus and can understand how they will affect our customers and Support teams
```
Comment (wordlee) [inline Support.md:161]:
```suggestion
- I collaborate with other Octopus leaders to ensure I am aware of all major initiatives at Octopus and can understand how they will affect our customers and Support teams.
```

File: Support.md [hunk 10]
```diff
@@ -76,41 +110,67 @@ The Senior Support Engineer role builds upon the foundation of the Support Engin
...
+- I have a track record of identifying significant problems, inhibitors and opportunities, and for creating lasting, best-fit solutions that align our strategy and Support efforts
```
Comment (wordlee) [inline Support.md:162]:
```suggestion
- I have a track record of identifying significant problems, inhibitors and opportunities, and for creating lasting, best-fit solutions that align our strategy and Support efforts.
```

File: Support.md [hunk 11]
```diff
@@ -76,41 +110,67 @@ The Senior Support Engineer role builds upon the foundation of the Support Engin
...
+- I identify people and teams with the necessary skills, work out how to fill gaps, and guide and influence these people and teams collaboratively
+- I use a variety of tools and techniques to effectively communicate short and long-term plans and goals to my teams, stakeholders, Octopus senior leadership, and the broader organization
+- I use my experience to collaboratively steer teams toward good long-term decisions
+- The results of the initiatives I champion are measurable
+
+### I share my knowledge both within the support team, and with the Octopus and wider community
+- I actively influence support teams across Octopus resulting in improved support excellence and overall productivity
+- I actively talk both internally and externally about the lessons we are learning as we improve and expand the support team to clarifying our thinking, reinforce our learning pathways, and to attract customers and potential future employees
+- I have a proven track record of expertly tailoring my communication to the target audience with a result of increased clarity and impact
```
Comment (wordlee) [inline Support.md:176]:
```suggestion
- I actively influence support teams across Octopus resulting in improved support excellence and overall productivity.
- I actively talk both internally and externally about the lessons we are learning as we improve and expand the support team to clarifying our thinking, reinforce our learning pathways, and to attract customers and potential future employees.
- I have a proven track record of expertly tailoring my communication to the target audience with a result of increased clarity and impact.
```

File: Support.md [hunk 12]
```diff
@@ -1,30 +1,67 @@
 # Support Roles
 
+## Junior Support Engineer
+
+_I am starting my journey at Octopus. I am an enthusiastic learner, I show great potential and have a high growth trajectory toward independent contribution._
+
+
+>We avoid distinguishing too much between Junior Support Engineer and Support Engineer. These roles are primarily focused on learning with the aim to grow into the contribution of a Senior Support Engineer or beyond. The primary indicator inside these levels is how far you are along your journey from dependent contribution to independent contribution.
```
Comment (wordlee) [inline Support.md:8]: I don't know if this distinction is helpful or not, but I've always thought of the difference between "junior role" and "role", as a junior being somebody who is actively learning how to do the role and needs support, training, feedback, mentoring, while somebody in a non-junior role also needs those to continue to grow, they don't need it as much to be independently productive.

**Review by wordlee — APPROVED** [review]
I pushed some minor line edits and left a couple of comments. Looks good.

**Pull request merged**